### PR TITLE
Attempt to use github workflow

### PR DESCRIPTION
**Pull request by simoncozens**
As mentioned in #6. Uses bits from Pillow-wheel and existing CI build.

### CODE REVIEW

File: .github/workflows/build.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
+if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
```
Comment (anthrotype) [inline .github/workflows/build.sh]: looks like this `build.sh` shell script is no longer used after you switched from muiltibuild to cibuildwheel

File: .github/workflows/wheels.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+name: Wheels
+
+on: [push, pull_request]
+
+env:
+  REPO_DIR: .
+  BUILD_DEPENDS: ""
+  TEST_DEPENDS: "pytest pytest-cov"
+  WHEEL_SDIR: wheelhouse
```
Comment (anthrotype) [inline .github/workflows/wheels.yml]: I think these environment variables are for a multibuild setup, not for cibuildwheel, you should try see what are the equivalent ones that cibuildwheel reads

**Review by anthrotype — COMMENTED** [review]
thanks Simon for taking a stab at this. Not sure why the configure step fails in https://github.com/fonttools/ttfautohint-py/pull/8/checks?check_run_id=2831985491, it says "compiler cannot create executable", maybe PATH is messed up somehow. Have you got a local M1 machine to test things with? Unfortunately I don't

Re: multibuild vs cibuildwheel, I've used the former more than the latter, but cibuildwheel now seems to be the new official way to build python wheels (moved under PyPA org) so whatever works and is easier to set up/maintain.
One thing I don't like is that cibuildwheel runs multiple builds one after another in a single runner, so it's slower than running each in parallel one distinct runners. There's probably some clever github actions matrix setup to allow running one build each on distinct, parallel runners

**Comment by simoncozens** [comment]
The reason the OS X build is not currently working is that it it's trying to build universal wheels. This works locally on OS X 11.0 but not with the 10.5 toolchain. I have filled in the form to get access to the 11.0 GitHub runners, but we don't have that yet. I don't have an M1 machine myself but Eli has installed the wheels I generated at home.

**Comment by simoncozens** [comment]
@anthrotype: We now have access to the macOS 11 runner, so can generate universal wheels. These work on ARM and x86 macOS, and all Python versions. This also generates the Linux wheels. I am not sure why Windows can't find the git executable; I have tried various things to set the path or give access to the gnulib sources, but no dice.

I think it would be good to get at least the macOS wheels deployed ASAP, as M1 users currently can't install anything which uses ttfautohint (gftools builder, Google Fonts template repo, etc.).

**Comment by anthrotype** [comment]
> I am not sure why Windows can't find the git executable; I have tried various things to set the path or give access to the gnulib sources, but no dice.

I don't know. Does GHA environment provide mingw-w64 toolchain and msys2 bash like appveyor does? 
Maybe we can temporarily keep Appveyor for the windows wheels and only use GHA for the linux/mac ones? I did that for other projects as well. As long as Appveyor continues to work and is free

**Comment by simoncozens** [comment]
This is ready to merge now, I think. I guess the next step would be to auto-upload the wheels to pypi on release.

**Comment by anthrotype** [comment]
> auto-upload the wheels to pypi on release.

done with https://github.com/fonttools/ttfautohint-py/commit/5bda1e3de062bdd93d78ff6ba416084c0afbcf7d

I'll try to tag a pre-release shortly and see if it works

**Pull request merged**